main.go

```diff
@@ -176,6 +178,7 @@ func main() {
 		config.WithLogger(ctrl.Log.WithName("config")),
 		config.WithVersion(v),
 		config.WithCollectorImage(collectorImage),
+		config.WithCreateRBACPermissions(createRBACPermissions),
```

Comment:
suggestion (take it or leave it): we could incorporate the new permissions checking capability to see if the service account for the operator has access to create RBAC permissions. I'm unsure if that would go in the operator logs, collector webhook, etc. What do you think?

Reply:
But that check will be always `true` since we are adding the permissions as part of the bundle, right?

Reply:
not really. This is my confusion as to why we're adding this permission to the bundle in the first place. Aren't we adding this RBAC permission to the serviceaccount of the collector not the operator? We shouldn't need the operator to have this RBAC permission because the operator isn't doing any operations on the config.openshift.io group, right?

Reply:
You cannot assign permissions over one service account if the service account has no permission over the resource. In other words: the operator cannot give permissions to the collector service account for the config.openshift.io group if the service account of the operator has no permissions over config.openshift.io group.

Reply:
ahhh i see, thank you very much for explaining! Now i understand.